packages/@aws-cdk/aws-ecs/lib/load-balanced-fargate-service.ts

```diff
@@ -121,8 +128,11 @@ export class LoadBalancedFargateService extends cdk.Construct {
       cpu: props.cpu
     });
 
+    const optIn = props.createLogDriver !== undefined ? props.createLogDriver : true;
+
     const container = taskDefinition.addContainer('web', {
       image: props.image,
+      logging: optIn === true ? this.addLogDriver(this.id) : undefined
```

Comment:
You don't have to compare a boolean to `true` to test it. You can just go:

```
logging: optIn ? this.addLogDriver(this.id) : undefined
```

Reply:
oops. done.